python/ray/serve/_private/http_proxy.py

```diff
@@ -56,6 +58,7 @@
     "RAY_SERVE_HTTP_REQUEST_MAX_RETRIES cannot be negative."
 )
 
+TIMEOUT_ERROR_CODE = "timeout"
```

Comment:
normally, the error code normally is a string of number? Is the timeout error code exposing to the users?